source/monsters/monsters_a-z/A/index.rst

```diff
@@ -5,5 +5,5 @@ A
    :titlesonly:
    :glob:
 
-   */index
+   **/index
```

Comment:
It's different depending on whether you want all subdirectories or just one, I believe? In this case I wanted all of them.